backend/src/main/java/com/dhbw/tutorsystem/module/Module.java

```diff
@@ -0,0 +1,24 @@
+package com.dhbw.tutorsystem.module;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+public class Module {
```

Comment:
Why is the @Table annotation not necessary here?

Reply:
@table isn't needed anymore due to new application.properties